### PR TITLE
RD-1468 Blueprint labels docs

### PR DESCRIPTION
**Pull request by YoniItzhak**
Adding documentation about `labels` and `blueprint_labels` in blueprints.

### CODE REVIEW

File: content/developer/blueprints/spec-blueprint-labels.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: bt_wiki
+title: Blueprint Labels
+category: Blueprints
+draft: false
+weight: 600
+aliases: /blueprints/spec-blueprint-labels/
+---
+
+`blueprint-labels` are useful for automatically attach labels to the blueprint. The labels are saved in lowercase.
```
Comment (ofercloudify) [inline content/developer/blueprints/spec-blueprint-labels.md:10]:
```suggestion
`blueprint-labels` are used to tag the blueprint object (unlike `labels` which tag the deployment). `blueprint-labels` are used for automatically attaching labels to the blueprint. The labels are saved in lowercase. 
```

File: content/developer/blueprints/spec-labels.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+layout: bt_wiki
+title: Labels
+category: Blueprints
+draft: false
+weight: 600
+aliases: /blueprints/spec-labels/
+---
+
+`labels` are useful for automatically attach labels to each deployment that is deployed from the blueprint.
```
Comment (ofercloudify) [inline content/developer/blueprints/spec-labels.md:10]:
```suggestion
`labels` are used to tag the deployment object generated from the blueprint (unlike `blueprint-labels` which tag the blueprint object itself)
`labels` are useful for automatically attaching labels to each deployment that is deployed from the blueprint. 
```

**Review by ofercloudify — CHANGES_REQUESTED** [review]

File: content/developer/blueprints/spec-blueprint-labels.md (hunk 1 above)
Comment (YoniItzhak) [reply content/developer/blueprints/spec-blueprint-labels.md]: Thanks, changed.

File: content/developer/blueprints/spec-labels.md (hunk 2 above)
Comment (YoniItzhak) [reply content/developer/blueprints/spec-labels.md]: Thanks, changed.

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**